cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,77 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import ArticlePage from '../support/pages/article.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import faker from 'faker';
+
+const signInPage = new SignInPageObject();
+const homePage = new HomePageObject();
+const articlePage = new ArticlePage();
 
 describe('Article', () => {
-  before(() => {
+  let user;
 
-  });
+  let title;
+  let description;
+  let content;
+  let tags;
 
   beforeEach(() => {
     cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+
+      signInPage.visit();
+
+      cy.register(user.email, user.username, user.password);
+
+      signInPage.typeEmail(user.email);
+      signInPage.typePassword(user.password);
+      signInPage.clickSignInBtn();
+
+      homePage.clickNewArticle();
+
+      title = faker.lorem.words(2);
+      description = faker.lorem.words(3);
+      content = faker.lorem.words(4);
+      tags = 'tag1,tag2,tag3';
+
+      articlePage.fillTitle(title);
+      articlePage.fillDescription(description);
+      articlePage.fillContent(content);
+      articlePage.fillTags(tags);
+      articlePage.submitArticle();
+    });
   });
 
   it('should be created using New Article form', () => {
-
+    cy.get('.banner h1').should('contain', title);
+    cy.get('.article-content').should('contain', content);
   });
 
   it('should be edited using Edit button', () => {
+    cy.get('a.btn-outline-secondary span').contains('Edit Article').click();
+
+    const editedTitle = faker.lorem.words(2);
+    const editedDescription = faker.lorem.words(2);
+    const editedContent = faker.lorem.paragraphs(1);
+    const editedTag = ('mag1,mag2');
```

Comment:
There seems to be a typo in the 'editedTag' value. It should be an array of strings like 'tag1,tag2' as used in line 40, not 'mag1,mag2'.

Reply:
No, it's my edited data for edited article )